htdocs/web_portal/css/web_portal.php

```diff
@@ -1,3 +1,22 @@
+<?php 
+require_once __DIR__ . '/../../../lib/Gocdb_Services/Factory.php';
+header("Content-type: text/css"); 
+// Load variable values from local configuration //
+\Factory::getConfigService()->setLocalInfoOverride($_SERVER['SERVER_NAME']);
+
+$background_top = \Factory::getConfigService()->getBackgroundTop();
+$background_bottom = \Factory::getConfigService()->getBackgroundBottom();
+$header_text_colour = \Factory::getConfigService()->getHeadingTextColour();
+
+/* if (\Factory::getConfigService()->getDefaultScopeName() == 'EOSC-hub') {
```

Comment:
Is there a reason to include this commented out code?

Reply:
None known now. I've removed it.